src/components/animated/_shrink.less

```diff
@@ -0,0 +1,23 @@
+.shrink-btn {
+  &:hover {
+    display: inline-block;
+    vertical-align: middle;
+    -webkit-transform: perspective(1px) translateZ(0);
+    transform: perspective(1px) translateZ(0);
+    box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+    -webkit-transition-duration: 0.3s;
+    transition-duration: 0.3s;
```

Comment:
Can you put the rules related to `transition-duration` outside of hover? It will make the transition more seemless when you move your mouse outside of the button.

Reply:
I don't quite understand, do you want me to shift it to focus?

Reply:
No just put it under `.shrink-btn` on line 2. It makes the transition better. Move as well rules related to `transition-property`